CMakeLists.txt

```diff
@@ -675,9 +686,11 @@ function (config_with_llvm)
     quipper_perf
     status_provider
     symbol_map)
-  add_test(NAME llvm_propeller_program_cfg_proto_builder_test COMMAND llvm_propeller_program_cfg_proto_builder_test)
+  add_test(NAME llvm_propeller_program_cfg_proto_builder_test
```

Comment:
Can you revert the formatting only changes?

Reply:
Done.